content/SCALE/GettingStarted/Configure/FirstTimeLogin.md

```diff
@@ -103,8 +100,8 @@ If using the CORE **Update** UI option, log into SCALE with the CORE root user c
 {{< include file="/static/includes/CreateAdminDisableRoot.md" >}}
 {{< /expand >}}
 
-### Logging In as Admin
-After setting up the admin user from one of the scenarios documented above, enter **admin** and the password to log into SCALE.
+### Logging In as Truenas_Admin
```

Comment:
Could we make this more generic to be more consistent with some of the other generic section headers and avoid having to update later if/when the default name changes again?

```suggestion
### Logging In as the Default Administrator
```

Reply:
The default admin is truenas_admin, and I wanted to be explicit about this as we are used to just admin.